### ISSUE TITLE
Data-Mining-Review

### ISSUE THREAD

**Issue by ZhihuangLi** [ZhihuangLi/DataScience-blog#1]
Content:
1. Introduction
- What is data mining?
- Origins of data mining
- Data Mining tasks
2. Data
- Types of data
- Data quality
- Data preprocessing
- Measures of similarity and dissimilarity
3.  Classification
- Basic concepts
- General framework for classification
- Decision tree classifier: entropy, Gini index, gain
4. kNN
- naive Bayes: conditional probability
5. Logistic regression
- gradient descent algorithm
6. Linear SVM
- the decision boundary
7. Neural networks
- Representation
- learning
8. k-means
9. Association analysis
- support, confidence, association rule
